nixos/modules/services/monitoring/netdata.nix

```diff
@@ -201,7 +201,7 @@ in {
       serviceConfig = {
         ExecStart = "${cfg.package}/bin/netdata -P /run/netdata/netdata.pid -D -c /etc/netdata/netdata.conf";
         ExecReload = "${pkgs.util-linux}/bin/kill -s HUP -s USR1 -s USR2 $MAINPID";
-        ExecPostStart = ''while [ "$(netdatacli ping)" != pong ]; do sleep 0.5; done'';
+        ExecStartPost = ''while [ "$(netdatacli ping)" != pong ]; do sleep 0.5; done'';
```

Comment:
This broke netdata compeletely:
`systemd[3405029]: netdata.service: Failed at step EXEC spawning while: No such file or directory`